Doc/library/xml.etree.elementtree.rst

```diff
@@ -816,16 +816,25 @@ Functions
    loader fails, it can return None or raise an exception.
 
 
-.. function:: xml.etree.ElementInclude.include( elem, loader=None)
+.. function:: xml.etree.ElementInclude.include( elem, loader=None, base_url=None, \
+                                                max_depth=DEFAULT_MAX_INCLUSION_DEPTH)
```

Comment:
Not sure if it's better to use the (undocumented) named constant or the integer here. I wouldn't want the constant to be documented (it can't be changed, for example), and the actual value is probably best looked up through introspection (since it might change). So, just having a talking name here is probably best.

Reply:
Thanks for the spacing fixes!

By talking name, do you mean use lowercase and spaces? I couldn't really find an example in the docs to match against. (Also, feel free to make more changes directly if you find that efficient!)

Reply:
With "talking name", I meant a named constant whose name explains the purpose.
I otherwise changed my mind, though. It's better if it's clear in the docs what the maximum depth is, even if that duplicates the value from the code. We should show the integer value here. Users will look up the documentation way more often than we risk to change the default value.

Reply:
Sounds good, made the change!